src/custom/assets/cow-swap/Ableton Folder Info/dc66a3fa-0fe1-5352-91cf-3ec237e9ee90.xmp

```diff
@@ -0,0 +1,17 @@
+<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="XMP Core 5.6.0">
```

Comment:
what´s this file??

Reply:
Not sure how it got there and when. But removed in https://github.com/cowprotocol/cowswap/pull/2439